content/guides/cloud/smart-orchestration.md

```diff
@@ -91,56 +96,61 @@ changes.
 Canceling an **entire** test run, even if parallelized, upon the first test
 failure will:
 
-1. **Reduce CI costs**. These cost savings can be significant for large test
+1. **Save time**. Resolve test outcomes faster.
+2. **Reduce CI costs**. These cost savings can be significant for large test
    suites.
-2. **Quickly free-up CI resources** for use by other members of a team.
-3. **Ensure availability of CI resources** for quick validation of the failure's
-   fix.
+3. **Free-up CI resources** for validating fixes, and helping other users.
 
 </Alert>
 
-### Toggle cancellation of test runs upon first failure
+### Activate Auto Cancellation in Cypress Cloud
 
 <Alert type="success">
 
 <strong class="alert-header"><Icon name="star"></Icon> Premium Cypress Cloud
 Feature</strong>
 
-**Canceling test runs when a test fails** is a _Smart Orchestration_ feature
-available to users with a
-[Business Cypress Cloud plan](https://cypress.io/pricing).
+**Auto Cancellation** is a _Smart Orchestration_ feature available to users with
+a [Business or Enterprise tier Cypress Cloud plan](https://cypress.io/pricing).
+Wondering if upgrading your Team tier subscription is worth it? Check out the
+[recommendations](/guides/cloud/runs#Recommendations) on the run overview page
+to see how much time we predict these features would save you.
 
 </Alert>
 
-Controlling cancellation of test runs upon the first failed test is a _Smart
-Orchestration_ feature that is managed within a project's settings.
+Auto Cancellation is a _Smart Orchestration_ feature that is managed within a
+project's settings. Coming soon, you will also have the option to enable the
+feature on a per-run basis with the Cypress CLI.
 
 <Alert type="info">
 
 <strong class="alert-header">Consideration for Teams</strong>
 
 If your development, testing, or QA teams operate in a highly collaborative
 workflow where multiple people are working on various test failures at the same
-time, it may be helpful to disable run cancellation upon first failure, so **all
-failing tests can be surfaced for each test run**.
+time, it may be helpful to increase the failures threshold or even disable Auto
+Cancellation altogether, so multiple failing tests can be surfaced for each test
+run.
 
 </Alert>
 
 To enable or disable this feature:
 
 1. Select the desired project within Projects view to visit Cypress Cloud.
 2. Click "Project Settings" with the right-hand sidebar.
-3. Scroll to the Smart Orchestration section within Project Settings page
-4. Toggle "Cancel run when a test fails."
+3. Scroll to the Smart Orchestration section within Project Settings page.
+4. Toggle "Auto Cancellation".
+5. Set the "failures before auto canceling" threshold (or just leave it at the
+   recommended value).
```

Comment:
```suggestion
   default value of 1).
```
This isn't necessarily a "recommended" value; we recognize this is case-by-case.